pygfunction/ground_heat_exchanger.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Union, Dict
+
+import numpy.typing as npt
+
+from .borefield import Borefield
+from .enums import PipeType
+from .media import Fluid
+from .networks import Network
+from .pipes import get_pipes
+
+
+class GroundHeatExchanger:
+
+    def __init__(self,
+                 H: npt.ArrayLike,
+                 D: npt.ArrayLike,
+                 r_b: npt.ArrayLike,
+                 x: npt.ArrayLike,
+                 y: npt.ArrayLike,
+                 pipe_type: PipeType,
+                 pos: List[tuple],
+                 r_in: Union[float, tuple, list],
+                 r_out: Union[float, tuple, list],
+                 k_s: float,
+                 k_g: float,
+                 k_p: float,
+                 fluid_name: str,
+                 fluid_concentration_pct: float,
+                 m_flow_ghe,
```

Comment:
Would prefer to not have to pass this in to the constructor, but that's going to require some more work.